config/bootstrap.php

```diff
@@ -12,36 +12,47 @@
  */
 
 use Doctrine\Common\Annotations\AnnotationReader;
-use Symfony\Component\ErrorHandler\Debug;
+use Symfony\Component\Dotenv\Dotenv;
 use Symfony\Component\Yaml\Yaml;
 
-require __DIR__ . '/../vendor/autoload.php';
-require __DIR__ . '/../src/RequirementChecker.php';
+require dirname(__DIR__) . '/vendor/autoload.php';
+require dirname(__DIR__) . '/src/RequirementChecker.php';
+
+// Load cached env vars if the .env.local.php file exists
+// Run "composer dump-env prod" to create it (requires symfony/flex >=1.2)
```

Comment:
requried flex? :disappointed:

Reply:
Supporting the option does not mean that we have to use Flex to assemble packages. Let's worry about other things first.